src/compositor/windowpixmapitem.cpp

```diff
@@ -556,6 +556,7 @@ QSGNode *WindowPixmapItem::updatePaintNode(QSGNode *oldNode, UpdatePaintNodeData
             m_textureProvider = prov;
 
             prov->fbo = 0;
+            prov->t = 0;
```

Comment:
I guess it doesn't change much. I usually don't implement constructors for small helper classes instantiated in a unique place, but i can.